Framework/DataHandling/src/SetSample.cpp

```diff
@@ -558,11 +559,15 @@ const Geometry::SampleEnvironment *
 SetSample::setSampleEnvironmentFromFile(API::ExperimentInfo &exptInfo, const Kernel::PropertyManager_const_sptr &args) {
   using Kernel::ConfigService;
 
-  const std::string envName = args->getPropertyValue(SEArgs::NAME);
+  std::string envName = args->getPropertyValue(SEArgs::NAME);
+  boost::algorithm::trim(envName);
```

Comment:
I'm not sure I quite follow what's happening here? There are now quotes coming through here?